Readability.js

```diff
@@ -2033,11 +2033,10 @@ Readability.prototype = {
       }
 
       var weight = this._getClassWeight(node);
-      var contentScore = 0;
 
       this.log("Cleaning Conditionally", node);
 
-      if (weight + contentScore < 0) {
```

Comment:
I see that `contentScore` is never set to anything other than `0`, but was it supposed to be set to something? Or is the `weight` the same as the "content score"?

Reply:
I would assume this comment `// TODO: Consider taking into account original contentScore here.` means `contentScore` could/should potentially be based on `node.readability.contentScore`.

Reply:
Using the node's contentScore breaks a lot of tests so there's probably something that was to be done there. Should I revert this change?

Reply:
I'd leave it like this for now, yes please. It might be worth a separate investigation as to why tests break and if we should either remove both this code and the TODO comment, and add an explanation of why we can't/shouldn't use the `contentScore`, or fix it so we *do* use it.